.gitignore

```diff
@@ -37,3 +37,9 @@ artifacts-zk
 cache-zk
 deployments-zk
 deployments
+
+generated
+
+# artifacts
+temp_repo
+packages/artifacts/src/abis/abis.ts
```

Comment:
Why `packages/artifacts/src/abis/abis.ts` and not just `packages/artifacts/*.ts`? Also, NodeJs will not work with TS files, we will need rollup or something to strip down the types for plain JS artifacts.